apps/web/pages/index.vue

```diff
@@ -21,6 +21,7 @@
         >
           <button
             v-if="experimentalAddBlock && disableActions && isPreview"
```

Comment:
```suggestion
            v-if="disableActions && isPreview"
```
This is what I meant by removing the flag completely. And then in `nuxt.config.ts` it can be removed as well.